src/controllers/update/update.manage.trusts.orchestrator.controller.ts

```diff
@@ -60,6 +69,60 @@ const shouldGoToReviewTheTrust = (trustInReview: Trust) => {
   return !trustInReview.review_status?.reviewed_trust_details;
 };
 
+export const trustChangeHandler = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
these changes are unrelated  to CYA? Do you just need to merge with main?

Reply:
no these changes were introduced by me to handle the change links me and Cory just have some overlap